adminSiteServer/appClass.tsx

```diff
@@ -154,8 +154,18 @@ export class OwidAdminApp {
         // error handler if it exists.
         if (bugsnagMiddleware) app.use(bugsnagMiddleware.errorHandler)
 
-        // todo: we probably always want to have this, and can remove the isDev
-        if (this.options.isDev) app.use("/", mockSiteRouter)
+        if (this.options.isDev) {
+            // https://vitejs.dev/guide/ssr
+            const { createServer } = await import("vite")
```

Comment:
@danyx23 dynamic imports currently don't work with Jest. Should I try to use their experimental support for ESM or is adding `isTest` option and skipping the import in that case ok?

Reply:
We don't need the mock site router atm, the only thing the dbtests care about is that the api REST endpoints can be reached - so I think guarding this with an isTest that we set when we run `make dbtest` should be fine.